commcare_connect/templates/opportunity/payments.html

```diff
@@ -0,0 +1,22 @@
+{% extends "opportunity/worker_list_base.html" %}
+{% load i18n %}
+
+{% block tab_options %}
+  <div x-data="{showPaymentImportModal: false, showPaymentExportModal: false}">
+      <button type="button" class="button-icon" @click="showPaymentImportModal= true"
+              {% if request.org_membership.is_viewer %} disabled {% endif %}
+      >
+        <i class="fa-solid fa-upload text-brand-deep-purple"></i>
+      </button>
+      <button type="button" class="button-icon" @click="showPaymentExportModal = true"
+              {% if request.org_membership.is_viewer %} disabled {% endif %}
+      >
+        <i class="fa-solid fa-download text-brand-deep-purple"></i>
+      </button>
+      {% url 'opportunity:payment_import' request.org.slug opportunity.pk as payment_import_url %}
+      {% include "components/worker_page/import_modal.html" with modal_name="showPaymentImportModal" modal_title=_("Import Payment Records") import_url=payment_import_url input_name="payments" input_help_text=_('The file must contain at least the "Username", "Amount" and "Payment Date" columns.') %}
```

Comment:
Is there a reason these help texts are sometimes in the python view and sometimes in the template?